_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -36295,22 +36341,27 @@
 /obj/machinery/atmospherics/pipe/manifold4w/scrubbers,
 /turf/open/floor/plasteel/white,
 /area/medical/medbay/central)
-"bLY" = (
-/obj/machinery/atmospherics/components/unary/vent_pump/on,
-/turf/open/floor/plasteel/white,
-/area/medical/virology)
 "bLZ" = (
 /obj/structure/cable{
 	icon_state = "1-2"
 	},
 /turf/open/floor/plasteel,
 /area/hallway/primary/aft)
 "bMa" = (
-/obj/machinery/atmospherics/pipe/simple/supply/hidden{
-	dir = 5
+/obj/structure/closet/crate/freezer,
+/obj/item/organ/tongue/zombie,
+/obj/item/organ/tongue/robot{
+	pixel_x = -2;
+	pixel_y = -7
 	},
-/turf/open/floor/plasteel/white,
-/area/medical/virology)
+/obj/item/organ/tail/lizard,
+/obj/item/organ/lungs/plasmaman,
+/obj/item/organ/heart/freedom,
```

Comment:
I hope you realize how OP this is.

Reply:
I checked out everything I added, my thoughts on that and a couple others are that its not a big deal and ill remove it if it becomes a big deal.

I see it as a Hearty Punch except instead of lasting for 3 minutes it only procs once/heals once. Ephedrine gives you -1 speed but if your at <20hp (which is what youre at when it triggers) you're still just a slow ass bitch who just needs another hit or two to crit for good.